### ISSUE TITLE
Add milestone board to help manage top level tasks

### ISSUE THREAD

**Issue by robchandhok** [codetree/feedback#111]
One thing we are missing as we try codetree is a top level view of our milestones, which we used to keep in another board-ish app.  Since codetree already has the ability to annotate milestones with ordering, if you could add attributes to milestones in codetree and use them for backlog/active/etc. it would be very powerful.

**Comment by derrickreimer** [comment]
@robchandhok this is an interesting idea. To clarify, you'd like to have a board that has one column per milestone? Could you explain a bit more about the "add attributes to milestones" part?

**Comment by robchandhok** [comment]
No, it's more than that.  Consider issues.  With codetree I can layer things like "active" or other states on issues.  I would like to do the same thing for milestones - show them progress with more information than "open" or "closed"  Imagine a "task board" view of milestones instead of just a list.

Also would like to be able to see/edit the milestone description in that view.

This would be magical!

**Comment by kareem** [comment]
@robchandhok interesting idea.  Do you have a screenshot you could share of how you do this today?  Feel free to redact and email if you don't want it public - k@codetree.com.  Thanks.
